src/components/RichTextEditor/RichTextEditor.spec.tsx

```diff
@@ -157,27 +158,35 @@ describe("RichTextEditor Component", () => {
         cy.get("[contenteditable=true]").should("include.html", "text-align: right");
     });
 
-    it("renders headings", () => {
+    it("renders a heading", () => {
         mount(<RichTextEditor />);
 
         insertTextAndOpenToolbar();
         cy.get(TOOLBAR).should("be.visible");
         cy.get(TEXTSTYLE_DROPDOWN_TRIGGER).click({ force: true });
-
         cy.get(TEXTSTYLE_OPTION).first().click();
         cy.get("[contenteditable=true]").should("include.html", "<h1");
+    });
 
-        cy.get(TEXTSTYLE_DROPDOWN_TRIGGER).click({ force: true });
-        cy.get(TEXTSTYLE_OPTION).eq(1).click();
-        cy.get("[contenteditable=true]").should("include.html", "<h2");
+    it("renders a custom font", () => {
+        mount(<RichTextEditor />);
 
+        insertTextAndOpenToolbar();
+        cy.get(TOOLBAR).should("be.visible");
         cy.get(TEXTSTYLE_DROPDOWN_TRIGGER).click({ force: true });
-        cy.get(TEXTSTYLE_OPTION).eq(2).click();
-        cy.get("[contenteditable=true]").should("include.html", "<h3");
+        cy.get(TEXTSTYLE_OPTION).eq(5).click();
+        cy.get("[contenteditable=true]").should("include.html", textStyleClassnames[TextStyles.ELEMENT_CUSTOM2]);
+    });
 
-        cy.get(TEXTSTYLE_DROPDOWN_TRIGGER).click({ force: true });
-        cy.get(TEXTSTYLE_OPTION).eq(3).click();
-        cy.get("[contenteditable=true]").should("include.html", "<h4");
+    it("renders multiple editors", () => {
+        mount(
+            <>
```

Comment:
It might be worth testing that there is no breaking when using multiple editors, for example testing it by typing it into both and ensuring that the values remain different.

Reply:
Added